heron/instance/src/python/BUILD

```diff
@@ -7,8 +7,10 @@ pex_library(
     srcs = glob(['**/*.py']),
     deps = [
         '//heronpy/proto:proto-py',
-        "//heronpy/api:heron-python-py",
-        '//heron/common/src/python:common-py'
+        '//heron/common/src/python:common-py',
```

Comment:
can heronpy dep come first before the common?

Reply:
sure